labman/gui/templates/plate_list.html

```diff
@@ -91,14 +96,16 @@
     <option selected disabled>Choose plate type...</option>
     <option value='sample'>Sample plates</option>
     <option value='gDNA'>gDNA plates</option>
-    <option value='16S library prep'>16S library plates</option>
+    <option value='normalized gDNA'>Normalized gDNA plates</option>
+    <option value='16S library prep'>Amplicon library plates</option>
+    <option value='shotgun library prep'>Shotgun library plates</option>
   </select>
 </div>
 
 <table id="plateListTable" class="display" cellspacing="0" width="100%">
   <thead>
     <tr>
-      <th></th>
+      <th>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</th>
```

Comment:
Not blocking but will it be better to set the row width vs adding a bunch of spaces?

Reply:
Good point - modified it on the DataTable object declaration (which already had a value....)